Lesson6/isdigit.py

```diff
@@ -1,10 +1,33 @@
-num = input()
+def check_digit_in_string(n):
+    flag = True
```

Comment:
Oooops.. flag again? 😄 I don't even know.. Dmitry helped you or u helped Dmitry 🥲